ReleaseProcessAutomation.Tests/Steps/StartReleaseStepTests.cs

```diff
@@ -21,58 +21,132 @@
 using ReleaseProcessAutomation.Git;
 using ReleaseProcessAutomation.Steps;
 using ReleaseProcessAutomation.Steps.PipelineSteps;
+using Spectre.Console.Testing;
```

Comment:
Please don't do this sort of reformat in during a random PR. It makes for very hard reading. Is this just the one file that's using the wrong style or the first one with the new style?

if it's the first one with the new style, it would be best to have a single commit to fix all files.

if it's the only one with the old style, also a separate commit, but one that just fixes this file.